artem-serbin/README.md

```diff
@@ -0,0 +1,45 @@
+Google Maps
+===========
+
+
+
+Google Maps is a widely used web mapping service developed by Google. It provides users with detailed information about geographical locations and allows them to explore various places around the world. This README file provides an overview of Google Maps and its features.
+
+## Features
+* Interactive Maps: Google Maps offers interactive maps that allow users to zoom in and out, pan, and explore different areas. Users can view maps in different modes such as satellite view, terrain view, and street view.
```

Comment:
Advice: make each feature bold to highlight it from its description in the text.